buildscripts/kokoro/gae.cfg

```diff
@@ -0,0 +1,10 @@
+# Config file for internal CI
+
+# Location of the continuous shell script in repository.
+build_file: "grpc-java/buildscripts/kokoro/gae.sh"
+timeout_mins: 45
+action {
+  define_artifacts {
+    regex: "**/build/test-results/**/*.xml"
```

Comment:
This should probably be updated (or deleted) to fit what artifacts your test will generated. This can be done as your work on `gae.sh`